R/stan_betareg.fit.R

```diff
@@ -24,7 +24,8 @@ stan_betareg.fit <- function(x, y, z = NULL,
                              link.phi = NULL, ...,
                              prior = normal(), prior_intercept = normal(),
                              prior_z = normal(), prior_intercept_z = normal(),
-                             prior_ops = prior_options(), prior_PD = FALSE, 
+                             prior_dispersion = cauchy(0, 5),
+                             prior_ops = NULL, prior_PD = FALSE, 
```

Comment:
I meant not in `stan_betareg.fit` at all. `summarize_betareg_prior` will just need to be tweaked to get that information without needing `prior_ops`, similarly to how we changed  `summarize_glm_prior` to not require `prior_ops`.